.github/workflows/build.yml

```diff
@@ -490,6 +494,9 @@ jobs:
       run: ./ci-scripts/ci/publish-tfgen-package ${{ github.workspace }}
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        # See https://github.com/pulumi/scripts/pull/138/files
+        # Possible values: "all", "wheel".
+        PYPI_PUBLISH_ARTIFACTS: all
```

Comment:
Why isn't this set to "wheel" now we're publishing the wheels?

Reply:
So it turns out we have paying customers that rely on source-only installations, as hinted in https://github.com/pulumi/pulumi-kubernetes/issues/2540 we cannot simply remove source distributions and need to publish all for the moment to be backwards-compatible.